core/trino-main/src/main/java/io/trino/util/DateTimeUtils.java

```diff
@@ -61,10 +61,6 @@ private DateTimeUtils() {}
 
     public static int parseDate(String value)
     {
-        // in order to follow the standard, we should validate the value:
-        // - the required format is 'YYYY-MM-DD'
-        // - all components should be unsigned numbers
-        // https://github.com/trinodb/trino/issues/10677
```

Comment:
I believe that the comment should stay.
Note that this is not a TODO (the corresponding issue acts as a TODO) but rather a documentation of the current state. I think it's always good to document wherever we diverge from the standard.